src/main/java/org/folio/rest/impl/base/AbstractEmail.java

```diff
@@ -263,41 +285,50 @@ protected Future<EmailEntries> findEmailEntries(int limit, int offset, String qu
 
   protected Future<Void> deleteEmailsByExpirationDate(String expirationDate, String emailStatus) {
     Promise<Void> promise = Promise.promise();
+    logger.debug("deleteEmailsByExpirationDate:: parameters expirationDate: {}, emailStatus: {}", expirationDate, emailStatus);
     storageService.deleteEmailEntriesByExpirationDateAndStatus(tenantId, expirationDate, emailStatus,
       result -> {
         if (result.failed()) {
+          logger.warn("deleteEmailsByExpirationDate:: Failed to delete emails with expiration date {}", expirationDate);
```

Comment:
should result.cause() be logged as well?

Reply:
updated, thanks